Feature/esimw 501 add initiator cancel pending doc policy

athill opened the pull request.

Commit:
[#ESIMW-501] Added INITIATOR_CANCEL_PENDING policy and updated CancelAction to respect it

athill committed:
ESIMW-501: Add INITIATOR_CANCEL_PENDING doc policy

- CancelActionTest: Add class to test new functionality in CancelAction

jwbennet commented on rice-middleware/impl/src/test/java/org/kuali/rice/kew/actions/CancelActionTest.java:
```diff
@@ -0,0 +1,100 @@
+
+    private List<ActionRequestValue> actionRequestValues;
+
+    @Before public void setUp() {
```
The test annotations should be on their own lines

athill replied:
I was following conventions from other tests in foundation Rice, but will move all annotations to their own line, as I prefer.

athill replied:
Fixed. Thanks.

athill committed:
ESIMW-501: Add INITIATOR_CANCEL_PENDING doc policy

- CancelActionTest: Move annotations to their own lines.

athill committed:
ESIMW-501: Add INITIATOR_CANCEL_PENDING doc policy

- Add INITIATOR_CANCEL_PENDING to `PolicyNameType` enumeration in DocumentType.xsd
